app/models/order.rb

```diff
@@ -55,7 +55,7 @@ def send_charge_receipts
     return unless uses_charge_receipt?
 
     successful_charges.each do
-      SendChargeReceiptJob.set(queue: _1.purchases_requiring_stamping.any? ? "default" : "critical").perform_async(_1.id)
+      SendChargeReceiptJob.set(queue: "critical").perform_async(_1.id)
```

Comment:
Because stamping moved to download time, receipts no longer depend on stamping being finished.